packages/snackbar/src/mwc-snackbar-base.ts

```diff
@@ -85,24 +85,36 @@ export class SnackbarBase extends BaseElement {
     return {
       ...addHasRemoveClass(this.mdcRoot),
       announce: () => util.announce(this.labelElement),
-      notifyClosed: (reason: String) => {
+      notifyClosed: (reason: string) => {
         this.isOpen = false;
         this.dispatchEvent(new CustomEvent(CLOSED_EVENT, {
           bubbles: true,
           cancelable: true,
+          /* eslint-disable
+             @typescript-eslint/no-angle-bracket-type-assertion,@typescript-eslint/no-object-literal-type-assertion
+           */
           detail: <MDCSnackbarCloseEventDetail> {
             reason: reason
           }
+          /* eslint-enable
+             @typescript-eslint/no-angle-bracket-type-assertion,@typescript-eslint/no-object-literal-type-assertion
+           */
         }));
       },
-      notifyClosing: (reason: String) =>
-          this.dispatchEvent(new CustomEvent(CLOSING_EVENT, {
-            bubbles: true,
-            cancelable: true,
-            detail: <MDCSnackbarCloseEventDetail> {
-              reason: reason
-            }
-          })),
+      notifyClosing: (
+          reason: string) => this.dispatchEvent(new CustomEvent(CLOSING_EVENT, {
+        bubbles: true,
+        cancelable: true,
+        /* eslint-disable
+           @typescript-eslint/no-angle-bracket-type-assertion,@typescript-eslint/no-object-literal-type-assertion
+         */
+        detail: <MDCSnackbarCloseEventDetail> {
```

Comment:
Same here, move the type variable to the constructor.

Reply:
Done! https://github.com/material-components/material-components-web-components/pull/371/commits/f249f6242802a0e8f6960c867ddc32849a63ccad